service/src/test/java/com/ge/predix/acs/audit/PredixEventProcessorTest.java

```diff
@@ -0,0 +1,45 @@
+package com.ge.predix.acs.audit;
+
+import static org.mockito.Matchers.anyObject;
+
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import com.ge.predix.audit.sdk.AuditClient;
+import com.ge.predix.audit.sdk.exception.AuditException;
+import com.ge.predix.audit.sdk.message.AuditEventV2;
+import com.ge.predix.eventhub.EventHubClientException;
+
+public class PredixEventProcessorTest {
+
+    @InjectMocks
+    private PredixEventProcessor eventProcessor;
+
+    @Mock
+    private AuditClient mockedClient;
+
+    @Mock
+    private PredixEventMapper mockedMapper;
+
+    @BeforeMethod
+    public void setup() throws AuditException, EventHubClientException {
+        this.eventProcessor = new PredixEventProcessor();
```

Comment:
Contrary to what we thought, this line to instantiate `this.eventProcessor` doesn't seem necessary (runs without a `NullPointerException` for me)